hack/update-generated-runtime-dockerized.sh

```diff
@@ -40,24 +42,34 @@ if [[ -z "$(which protoc)" || "$(protoc --version)" != "libprotoc 3."* ]]; then
 fi
 
 function cleanup {
-	rm -f "${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go.bak"
-	rm -f "${KUBE_REMOTE_RUNTIME_ROOT}/api.pb.go.tmp"
+	for v in "${runtime_versions[@]}"; do
+		rm -f "${KUBE_REMOTE_RUNTIME_ROOT}/${v}/api.pb.go.bak"
+		rm -f "${KUBE_REMOTE_RUNTIME_ROOT}/${v}/api.pb.go.tmp"
+	done
 }
 
 trap cleanup EXIT
 
 gogopath=$(dirname "$(kube::util::find-binary "protoc-gen-gogo")")
 
-PATH="${gogopath}:${PATH}" \
-  protoc \
-  --proto_path="${KUBE_REMOTE_RUNTIME_ROOT}" \
-  --proto_path="${KUBE_ROOT}/vendor" \
-  --gogo_out=plugins=grpc:"${KUBE_REMOTE_RUNTIME_ROOT}" "${KUBE_REMOTE_RUNTIME_ROOT}/api.proto"
+function generate_code() {
+	RUNTIME_API_VERSION="$1"
+	KUBE_REMOTE_RUNTIME_PATH="${KUBE_REMOTE_RUNTIME_ROOT}/${RUNTIME_API_VERSION}"
+	PATH="${gogopath}:${PATH}" \
```

Comment:
not sure how it works in bash. PATH is a global variable. first call will set the first version. Second - will have the first version already set and add a new runtime to the end. Would it pick up the second runtime at all?

Reply:
Moving out of the function.